cmake/kokkos_arch.cmake

```diff
@@ -433,8 +433,14 @@ CHECK_AMDGPU_ARCH(VEGA906 gfx906) # Radeon Instinct MI50 and MI60
 CHECK_AMDGPU_ARCH(VEGA908 gfx908)
 
 IF(KOKKOS_ENABLE_HIP AND NOT AMDGPU_ARCH_ALREADY_SPECIFIED)
-  MESSAGE(SEND_ERROR "HIP enabled but no AMD GPU architecture currently enabled. "
-                     "Please enable one AMD GPU architecture via -DKokkos_ARCH_{..}=ON'.")
+  FIND_PATH(ROCM_ENUMERATOR_PATH rocm_agent_enumerator)
```

Comment:
Why did you chose this over `find_program`

Reply:
No good reason